reports/Pennsylvania.md

```diff
@@ -180,6 +180,18 @@ id: pa-philadelphia-10
 * https://twitter.com/jpegjoshua/status/1267636972404736000
 * https://twitter.com/jpegjoshua/status/1267599264257015816
 
+### Police pepper spray protesters on their knees | June 1st
+
+Protesters are seen in the middle of the road on their knees with their hands up. Police proceed to indiscriminately pepper spray them.
```

Comment:
I'd mention the officer walking across the median, spraying people on the median and then spraying people below